src/ocean/time/StopWatch.d

```diff
@@ -17,6 +17,9 @@
 
```

Comment:
It is the reason why I removed the paragraph in the documentation claiming on Linux the time resolution would be 1µs at best.